apps/additional_apps.sh

```diff
@@ -28,76 +28,72 @@ whiptail --title "Which apps do you want to install?" --checklist --separate-out
 "PreviewGenerator" "(Pre-generate previews)   " OFF \
 "LDAP" "(Windows Active directory)   " OFF \
 "Talk" "(Nextcloud Video calls and chat)   " OFF \
-"SMB-mount" "(Connect to SMB-shares from your local network)   " OFF 2>results
-
-while read -r choice
-do
-    case "$choice" in
-        "Fail2ban")
-            clear
-            run_app_script fail2ban
-        ;;
-
-        "Adminer")
-            clear
-            run_app_script adminer
-        ;;
-
-        "Netdata")
-            clear
-            run_app_script netdata
-        ;;
-
-        "OnlyOffice")
-            clear
-            run_app_script onlyoffice
-        ;;
-
-        "Collabora")
-            clear
-            run_app_script collabora
-        ;;
-
-        "Bitwarden")
-            clear
-            run_app_script tmbitwarden
-        ;;
-
-        "FullTextSearch")
-            clear
-           run_app_script fulltextsearch
-        ;;
-
-        "PreviewGenerator")
-            clear
-           run_app_script previewgenerator
-        ;;
-
-        "LDAP")
-            clear
-	    print_text_in_color "$ICyan" "Installing LDAP..."
-            if install_and_enable_app user_ldap
-	    then
-	        msg_box "LDAP installed! Please visit https://subdomain.yourdomain.com/settings/admin/ldap to finish the setup once this script is done."
-	    else msg_box "LDAP installation failed."
-	    fi
-        ;;
-
-        "Talk")
-            clear
-            run_app_script talk
-        ;;
-
-        "SMB-mount")
-            clear
-            run_app_script smbmount
-        ;;
-
-        *)
-        ;;
-    esac
-done < results
-rm -f results
+"SMB-mount" "(Connect to SMB-shares from your local network)   " OFF 3>&1 1>&2 2>&3)
+
+case "$choice" in
+    *"Fail2ban"*)
+        clear
+        run_app_script fail2ban
+    ;;&
+
+    *"Adminer"*)
+        clear
+        run_app_script adminer
+    ;;&
+
+    *"Netdata"*)
+        clear
+        run_app_script netdata
+    ;;&
+
```

Comment:
What would you say? Shall we delete the empty rows here?

Reply:
Are you really working with the latest master? 

Yes please, delete emtpy lines.

Reply:
@enoch85 Yes, I am working with the latest master afaik.
and why is this resolved?

Reply:
>  I have removed all empty lines and unnecessary spaces